docs/configuration-v1_3-exp.md

```diff
@@ -190,6 +190,16 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
     * **_gid_** (integer): the group ID of the new group.
     * **_password_hash_** (string): the hashed password of the new group.
     * **_system_** (bool): whether or not the group should be a system group. This only has an effect if the group doesn't exist yet.
+* **_boot_device_** (object): describes the desired boot device configuration.
+  * **_layout_** (string): the disk layout of the target OS image. This is only needed if also specifying `luks` or `mirror`. Supported values are `aarch64`, `ppc64le`, and `x86_64`. Defaults to `x86_64`.
+  * **_luks_** (object): describes the clevis configuration for encrypting the root filesystem.
```

Comment:
I think we discussed this, but just to check: are you purposely ignoring `custom` here?

Reply:
Yes.  Use of the sugar is not mandatory, and users who want to deeply customize their LUKS (or RAID) configuration can write the low-level config stanzas directly.